sources/util.ts

```diff
@@ -56,6 +56,18 @@ export function isHTMLLegendElement(
 	return isElement(node) && node.tagName === "LEGEND";
 }
 
+export function isSVGElement(node: Node | null): node is SVGElement {
+	return isElement(node) && node.ownerSVGElement !== undefined;
```

Comment:
I was getting a ts error in my IDE: 

> Property 'ownerSVGElement' does not exist on type 'Element'.ts(2339)

I am not sure if this is the best way to look at the defined properties -- but on SVG elements, it is either defined or `null`. I figured elsewhere it would be `undefined`.